### ISSUE TITLE
Simplify ArcGIS Server service publishing process

### ISSUE THREAD

**Issue by elinz** [Esri/ops-server-config#10]
Currently, publishers have to create an .sd file for each service because the .sd files that are uploaded to the server are deleted after publishing. Was made aware that there is a registry setting for preventing the .sd file from being deleted. Applying this registry setting will allow publishers to publish directly to the staging server without having to stage the service in a .sd file; this will prevent common publishing mistakes, such as updating the service but forgetting to re-create the .sd file first.

"Email from Joanne Parkinson, 7/19/2013: This is the registry key (regedit needs to be run as the SOC account):
HKEY_CURRENT_USER\Software\ESRI\Server10.2\ArcToolbox\Settings

This is the value (DWORD), set it to 1 to avoid the deletion of the SDs
KeepPublishedSD (DWORD)

When set to 1 the publishSD tool won’t delete the SD from the admin uploads folder of the server. The sd in the input folder will always be deleted. There is no way to control this."

Basically, need to:
1) Set this registry setting on the staging server.
2) Update the publishing process document to reflect the fact that publishing no longer need to create the .sd file first; they can publish their service directly to the server.
3) Change permissions on the service definitions folder on the file server so users can no longer write service definitions to the folder.
4) Create a script to extract the latest .sd files for each published service.
